spiceaidocs/docs/components/data-connectors/odbc.md

```diff
@@ -28,11 +28,12 @@ In addition to the connection string, the following [arrow_odbc builder paramete
 
 | Parameter               | Type           | Description                                                                                                                                                                                      | Default                                           |
 |-------------------------|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------|
-| max_bytes_per_batch     | number (bytes) | Upper allocation limit for transit buffer.                                                                                                                                                       | `512_000_000`                                     |
-| max_num_rows_per_batch  | number (rows)  | Upper limit for number of rows fetched for one batch.                                                                                                                                            | `65536`                                           |
-| max_text_size           | number (bytes) | Upper limit for value buffers bound to columns with text values.                                                                                                                                 | Unset (allocates driver-reported max column size) |
-| max_binary_size         | number (bytes) | Upper limit for value buffers bound to columns with binary values.                                                                                                                               | Unset (allocates driver-reported max column size) |
-| enable_db2_length_quirk | bool           | If connecting to IBM DB2, enable a [quirk](https://docs.rs/arrow-odbc/latest/arrow_odbc/struct.Quirks.html#structfield.indicators_returned_from_bulk_fetch_are_memory_garbage) to avoid crashing.| `false`                                           |
+| odbc_sql_dialect     | string | Override what SQL dialect is used for the ODBC connection. Supports `postgresql`, `mysql`, `sqlite` or `databricks` values.   | Unset (auto-detected) |
```

Comment:
```suggestion
| sql_dialect     | string | Override what SQL dialect is used for the ODBC connection. Supports `postgresql`, `mysql`, `sqlite` or `databricks` values.   | Unset (auto-detected) |
```

Reply:
The parameter is `odbc_sql_dialect`. `sql_dialect` makes no effect?

Reply:
This was changed to `sql_dialect` in https://github.com/spiceai/spiceai/pull/2028

Reply:
bug fixed in https://github.com/spiceai/spiceai/pull/2050